EIPS/eip-7775.md

```diff
@@ -0,0 +1,102 @@
+---
+eip: 7775
+title: BURN opcode
+description: An opcode to burn native token at the given address
```

Comment:
I find the word "token" here confusing. This can easily be confused with an ERC20 token. As I understand it, the "native cryptocurrency" which the evm uses (so for Ethereum: ETH) is used as "native token" here.

Reply:
yup you got it